services/self_bootstrap.py

```
@@ -58,6 +68,108 @@ async def _find_self_company():
+            stale.append(company)
+    return stale
+
+
+async def _count_specialists(company_id: str) -> int:
+    """Return the number of provisioned specialists for a company."""
+    try:
+        from services.company_graph_store import get_company_graph_store
+        store = get_company_graph_store()
+        specialists = await store.list_specialists(company_id)
+        return len(specialists)
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)